tutorials/tts/FastPitch_Data_Preparation.ipynb

```diff
@@ -0,0 +1,1125 @@
+        "For training it is beneficial for us to *normalize* our features. The most standard approach is to apply **mean-variance normalization** so that each feature has a mean of 0 and variance of 1. To do this we need to compute the *dataset statistics* with the mean and variance of each feature.\n",
+        "\n",
+        "For TTS it also helps\n",
+        "*   Normalize features using speaker-level statistics\n",
+        "*   Use the *voiced_mask* to set the feature values of non-voiced audio frames to 0.\n",
+        "\n",
+        "Using the [compute_feature_stats.py](https://github.com/NVIDIA/NeMo/blob/main/scripts/dataset_processing/tts/compute_feature_stats.py) script we will compute the mean and variance of each feature for each speaker. The input to the script is the same [config file](https://github.com/NVIDIA/NeMo/blob/main/examples/tts/conf/feature/feature_44100.yaml) we used to compute the features."
+      ],
+      "metadata": {
+        "id": "O8GiAnAMCNeh"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "feature_stats_script = NEMO_SCRIPT_DIR / \"compute_feature_stats.py\"\n",
+        "\n",
+        "train_manifest_filepath = DATA_DIR / \"train_manifest.json\"\n",
+        "output_stats_path = DATA_DIR / \"feature_stats_44khz.json\"\n",
+        "\n",
+        "args = [\n",
+        "    f\"--feature_config_path={feature_config_path}\",\n",
+        "    f\"--manifest_path={train_manifest_filepath}\",\n",
+        "    f\"--audio_dir={audio_dir}\",\n",
+        "    f\"--feature_dir={feature_dir}\",\n",
+        "    f\"--stats_path={output_stats_path}\"\n",
+        "]\n",
+        "\n",
+        "run_script(feature_stats_script, args)"
+      ],
+      "metadata": {
+        "id": "DC4c1L3CxH-h"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "The output feature statistics file contains the pitch and energy mean and variance for the entire dataset (under the key `global`) and for each speaker in the dataset."
+      ],
+      "metadata": {
+        "id": "zos96yaoFho1"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "!head $output_stats_path"
+      ],
+      "metadata": {
+        "id": "fOz1cpIdFcG9"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# HiFi-GAN Training"
+      ],
+      "metadata": {
+        "id": "oRO842MUyODC"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Our standard FastPitch model is a two-part recipe consisting of the **FastPitch** acoustic model which predicts a mel spectrogram from text, and **HiFi-GAN** vocoder which predicts audio from the mel spectrogram.\n",
+        "\n",
+        "We will train HiFi-GAN first so that we can use it to help evaluate the performance of FastPitch as it is being trained.\n",
```

Comment:
most of our tutorials recommended to finetune a HiFiGan from one of pre-trained NGC HiFiGan model checkpoints. Could you please also adding a sentence mentioning that?